docs/advanced/miscellaneous.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Miscellaneous"
+summary: "Extra Stuff"
+draft: false
+toc: true
+sidebar_position: 5
+---
+
+## Using GDB with Linux AppImages (32 bit and 64 bit)
+
+This guide is useful if for example you'd like to debug a 32 bit crash but cannot build a 32 bit PCSX2.
+
+1. **Put your AppImage into a directory by itself**
+
+   We'll call the AppImage `PCSX2.AppImage` and then navigate your terminal to this directory..
+
+2. **Extract the contents of the AppImage**
+
+   You can do this by doing `./PCSX2.AppImage --appimage-extract`
+
+   A new directory called `squashfs-root` should have been created.
+
+3. **Run the AppImage normally**
+
+   `./PCSX2.AppImage`
+
+4. **Get the PID of the current PCSX2 instance**
+
+   You can do `pidof PCSX2`.
+
+   If there is more than one pid that shows up, you have more than one PCSX2 running!
+
+   We'll call your running pid `<PID>`
+
+5. **Attach GDB (during a fault or when PCSX2 is running normally, doesn't matter)**
+
+   Note: In my case I have to run gdb as `sudo`
```

Comment:
Avoid first person, get rid of "I" usage and keep third person perspective whenever possible.